contributing_to_docs/create_or_edit_content.adoc

```diff
@@ -18,6 +19,24 @@ required to contribute
 * Read and review the link:doc_guidelines.adoc[Documentation guidelines] topic
 to understand the basic guidelines for consistency
 
+== Understanding the contribution workflow
+
+The following diagram outlines the steps required to add content to the OpenShift documentation:
+
+image::osdocs-contribution-workflow.png[OpenShift documentation contribution workflow]
+
+After you have identified a documentation requirement and created a ticket, you can contribute to the documentation directly or the OpenShift documentation team can create the content.
```

Comment:
Should we use "OpenShift documentation team", "OpenShift Docs team", or something else when referring to our team in the docs?

Reply:
"OpenShift Docs" might make the team name stand out more, and aligns with the email list address. Though I could understand an argument for wanting to spell it out lowercase, too. :shrug:

Reply:
Thanks, I will update the references to "OpenShift Docs".